Dockerfile.rootless

```diff
@@ -51,7 +53,6 @@ RUN chown git:git /var/lib/gitea /etc/gitea
 
 COPY docker/rootless /
 COPY --from=build-env /go/src/code.gitea.io/gitea/gitea /usr/local/bin/gitea
```

Comment:
`COPY` supports chown as a flag, it may even perhaps support chmod, but I am unsure.

Reply:
it does support chmod only chown

Reply:
moved chown to COPY